addons/purchase/models/purchase.py

```diff
@@ -717,11 +717,12 @@ def _prepare_stock_moves(self, picking):
 
     @api.multi
     def _create_stock_moves(self, picking):
-        moves = self.env['stock.move']
+        moves = self.env['stock.move'].with_context({'recompute': False})
```

Comment:
I'm pretty sure this is a bad idea.  I think this completely overwrites the context, a lesson I've learnt the hard way.  What if there are settings in there like timezone, lang or date that are needed.  Better would be (notwithstanding whether the PR is appropriate)

`moves = self.env['stock.move'].with_context(recompute=False)`

Reply:
Thanks. Indeed. I changed that...